simple/stats/config.py

```diff
@@ -120,7 +129,32 @@ def database(self) -> dict:
     return self.data.get(_DATABASE_FIELD)
 
   def _input_file(self, input_file_name: str) -> dict:
-    return self.data.get(_INPUT_FILES_FIELD, {}).get(input_file_name, {})
+    # Exact match.
+    input_file_config = self._input_files_config.get(input_file_name, {})
+    if input_file_config:
+      return input_file_config
+
+    # Wildcard match
+    if input_file_name not in self._input_file_name_keys.keys():
```

Comment:
no need to do this "if" check since it's covered above?

Reply:
It is needed. The first time an input file is accessed and it is not individually specified, it will not be in the `self._input_file_name_keys` dict and this conditional will create and cache an entry for it. Subsequent calls will just use this entry.